packages/blade/src/components/Tour/TourMask.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable react/jsx-no-useless-fragment */
+import React from 'react';
+import type { FlattenSimpleInterpolation } from 'styled-components';
+import styled, { css, keyframes } from 'styled-components';
+import usePresence from 'use-presence';
+import type { TourMaskRect } from './types';
+import { useWindowSize } from '~utils/useWindowSize';
+import { makeSpace, useTheme } from '~utils';
+import { makeMotionTime } from '~utils/makeMotionTime';
+import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
+
+const scaleIn = keyframes`
+  from {
+    fill-opacity: 0;
+  }
+  to {
+    fill-opacity: 1;
+  }
+`;
+const fadeOut = keyframes`
+  from {
+    fill-opacity: 1;
+  }
+  to {
+    fill-opacity: 0;
+  }
+`;
+
+const AnimatedFade = styled.rect<{ animationType: FlattenSimpleInterpolation | null }>(
```

Comment:
We already have a Fade component. Can we not use that?

Reply:
Nope, we do not have a reusable Fade component.